package.json

```diff
@@ -1,27 +1,29 @@
 {
   "name": "@asciidoctor/gulp-asciidoctor",
-  "version": "2.1.1",
-  "description": "Gulp AsciiDoctor plugin",
+  "version": "2.2.0",
+  "description": "Gulp Asciidoctor plugin",
```

Comment:
My pathological obsessiveness for capitalization is thanking you :laughing:

Reply:
I recognized it just a few seconds before 😆